wandb/sdk/launch/builder/kaniko_builder.py

```diff
@@ -271,6 +272,13 @@ def build_image(
                 batch_v1, build_job_name, 3 * _DEFAULT_BUILD_TIMEOUT_SECS
             ):
                 raise Exception(f"Failed to build image in kaniko for job {run_id}")
+            try:
+                logs = batch_v1.read_namespaced_job_log(build_job_name, "wandb")
+                warn_failed_packages_from_build_logs(logs, image_uri)
+            except Exception as e:
+                wandb.termwarn(
+                    f"{LOG_PREFIX}Failed to get logs for kaniko job {build_job_name}: {e}"
+                )
```

Comment:
should we reraise this?

Reply:
I don't think so, the run will just go to the FAILED state if the container failed to build.